src/main/java/hudson/plugins/deploy/DeployPublisher.java

```diff
@@ -47,17 +55,43 @@ public DeployPublisher(List<ContainerAdapter> adapters, String war, String conte
         this.contextPath = contextPath;
     }
 
+    @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
```

Comment:
It is a valid warning.
Just check if the workspace is null and throw `AbortException`